Frontend/Controllers/Projects/RationaleController.cs

```diff
@@ -44,43 +43,49 @@ public async Task<IActionResult> Project(string urn, bool returnToPreview = fals
                 return View("ErrorPage", project.Error.ErrorMessage);
             }
 
-            var model = new RationaleViewModel {Project = project.Result, ReturnToPreview = returnToPreview};
-
-            return View(model);
+            var projectResult = project.Result;
+            var vm = new RationaleProjectViewModel
+            {
+                Urn = projectResult.Urn,
+                OutgoingAcademyName = projectResult.OutgoingAcademyName,
+                ReturnToPreview = returnToPreview,
+                ProjectRationale = projectResult.Rationale.Project
+            };
+            
+            return View(vm);
         }
 
         [ActionName("Project")]
         [HttpPost("rationale")]
-        public async Task<IActionResult> ProjectPost(string urn, string rationale, bool returnToPreview = false)
+        public async Task<IActionResult> ProjectPost(RationaleProjectViewModel vm)
         {
-            var project = await _projectsRepository.GetByUrn(urn);
+            var project = await _projectsRepository.GetByUrn(vm.Urn);
             if (!project.IsValid)
             {
                 return View("ErrorPage", project.Error.ErrorMessage);
             }
-
-            var model = new RationaleViewModel {Project = project.Result, ReturnToPreview = returnToPreview};
-
-            model.Project.Rationale.Project = rationale;
-
-            if (string.IsNullOrEmpty(rationale))
+            
+            if (!ModelState.IsValid)
             {
-                model.FormErrors.AddError("rationale", "rationale", "Enter the rationale for the project");
-                return View(model);
+                return View(vm);
             }
 
+            var model = new RationaleViewModel {Project = project.Result, ReturnToPreview = vm.ReturnToPreview};
```

Comment:
Don't think we need this line anymore..can use project result instead for line 77